packages/account/src/Components/account-limits/account-limits-turnover-limit-row.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { formatMoney } from '@deriv/shared';
+import AccountLimitsTableCell from './account-limits-table-cell.jsx';
+import AccountLimitsContext from './account-limits-context';
+
+type TAccountLimitsTurnoverLimitRow = {
+    collection: Array<any>;
+    title: string;
+};
+
+const AccountLimitsTurnoverLimitRow = ({ collection, title }: TAccountLimitsTurnoverLimitRow) => {
+    const { currency } = React.useContext(AccountLimitsContext);
+
+    if (!collection?.length) {
```

Comment:
if we says to TS, that collection is array, why we are checking here existing of length?

Reply:
I think having a fallback is OK. Typing will just tell the type but not enforce it. May be another way to do is assign a default `[]` value?